### DISCUSSION TITLE
Given a root orgId how can I get that organisation's sourcing org admin and contributor org admin details

### DISCUSSION THREAD

**Discussion by aks30** [Sunbird-Lern/Community#42] [Query]
Looking for some help on how can we get sourcing org admin and contributor org admin details as mentioned below given a root org id?

srcOrgAdminId,srcOrgAdminUserName,contributorOrgAdminId,contributorOrgAdminUserName

cc - @Shakthieshwari, @kiranharidas187 @pradeepkumarcm96, @githubpiyush, @vijiurs

**Comment by aks30** [comment]
> Looking for some help on how can we get sourcing org admin and contributor org admin details as mentioned below given a root org id?
> 
> srcOrgAdminId,srcOrgAdminUserName,contributorOrgAdminId,contributorOrgAdminUserName
> 
> cc - @Shakthieshwari, @kiranharidas187 @pradeepkumarcm96, @githubpiyush, @vijiurs

@reshmi-nair , @krgauraw, @amit-tarento

**Reply by vijiurs** [reply]
@reshmi-nair , request your help here,

**Reply by reshmi-nair** [reply]
@aks30 As of now there is no role search api where you can pass the orgid and get the list of users with roles. But user search api may give a similar result if you try with the correct filters.

**Reply by aks30** [reply]
Thank you @reshmi-nair it would be great if you can help with relevant filters we should use to get the above information

**Reply by reshmi-nair** [reply]
@aks30 Pls try this kind of filter:
{
    "request": {
        "filters": {
            "roles.role": "REPORT_ADMIN",
             "organisations.organisationId": "01347943538447155253"
        }
    }
}

**Reply by githubpiyush** [reply]
@reshmi-nair I have tried using this filters but I am not able to find any orgId.
What does REPORT_ADMIN role means? Is it a sourceAdmin?
